README.md

```diff
@@ -89,45 +89,84 @@ behavior you may find.
 
 The following configuration properties are common to all exporters:
 
-| System property          | Environment variable     | Purpose                                                                                            |
-|--------------------------|--------------------------|----------------------------------------------------------------------------------------------------|
-| otel.exporter            | OTEL_EXPORTER            | To select exporter e.g. `otlp,jaeger`. Defaults to `otlp`                                          |
+| System property | Environment variable | Purpose                                                                                                                                                 |
+|-----------------|----------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------|
+| otel.exporter   | OTEL_EXPORTER        | Exporter to be used, can be a comma-separated list to use multiple exporters. Currently does not support multiple metric exporters. Defaults to `otlp`. |
```

Comment:
Maybe worth linking to the below section from here

Reply:
is `otlp_span` supported? I get `WARN io.opentelemetry.javaagent.tooling.TracerInstaller - No otlp_span span exporter found` using latest 0.9.0 release.

Reply:
nevermind, I created https://github.com/open-telemetry/opentelemetry-java-instrumentation/issues/1401

Reply:
@malafeev oh no, thanks for noticing this! I removed `otlp_span` and `otlp_metric` from doc for now until we get that fixed in next release.

@anuraaga I moved the OTLP doc up to the top of the list since it's the default.